src/main/java/ch/bfh/bti7081/view/NewSeminarView.java

```diff
@@ -0,0 +1,157 @@
+package ch.bfh.bti7081.view;
+
+import ch.bfh.bti7081.model.manager.SeminarCategoryManager;
+import ch.bfh.bti7081.model.manager.SeminarManager;
+import ch.bfh.bti7081.model.seminar.Seminar;
+import ch.bfh.bti7081.model.seminar.SeminarCategory;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.datepicker.DatePicker;
+import com.vaadin.flow.component.formlayout.FormLayout;
+import com.vaadin.flow.component.html.H1;
+import com.vaadin.flow.component.html.Label;
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.icon.VaadinIcon;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.component.select.Select;
+import com.vaadin.flow.component.textfield.NumberField;
+import com.vaadin.flow.component.textfield.TextArea;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.component.timepicker.TimePicker;
+import com.vaadin.flow.data.binder.Binder;
+import com.vaadin.flow.data.binder.BinderValidationStatus;
+import com.vaadin.flow.data.binder.BindingValidationStatus;
+import com.vaadin.flow.router.Route;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Route(value = "newSeminar", layout = Layout.class)
+public class NewSeminarView extends VerticalLayout {
+
+    private H1 title = new H1("Seminar erstellen");
+
+    private TextField seminarTitle = new TextField("Seminar-Titel");
+    private Select<SeminarCategory> seminarCategory = new Select<SeminarCategory>();
+
+    private TimePicker seminarTime = new TimePicker("Zeit");
+    private DatePicker seminarDate = new DatePicker("Datum");
+    private FormLayout dateComposite = new FormLayout(seminarDate,seminarTime);
```

Comment:
Warum nicht nur ein "Baustein" für Datum und Uhrzeit? Gibt es keinen solchen in Vaadin, der dann evt ein Timestamp oder ähnlich liefert (am praktischsten wäre natürlich LocalDateTime...)

Reply:
Simpel: Es gibt keinen solchen Baustein. Ich habe in einem früheren Commit einen solchen erstellt, dieser passt sich jedoch nicht richtig in das Design ein. Deswegen bin ich dazu übergegangen die zwei Standardelemente zu verwenden, die auch einfach so funktionieren.